src/test/java/educonnect/logic/parser/AddCommandParserTest.java

```diff
@@ -178,7 +178,7 @@ public void parse_compulsoryFieldMissing_failure() {
         assertParseFailure(parser, VALID_NAME_BOB + STUDENT_ID_DESC_BOB + EMAIL_DESC_BOB + TELEGRAM_HANDLE_DESC_BOB,
                 expectedMessage);
 
-        // missing phone prefix
+        // missing student id prefix
         assertParseFailure(parser, NAME_DESC_BOB + VALID_STUDENT_ID_BOB + EMAIL_DESC_BOB + TELEGRAM_HANDLE_DESC_BOB,
```

Comment:
Is line 178 and 181 the same test case? Also looks like student id is present?

Reply:
NAME_DESC_BOB, EMAIL_DESC_BOB, etc. all have a prefix attached; i.e. `n/Bob` or `e/bob@gmail.com`

VALID_STUDENT_ID_BOB does not have a prefix; i.e. A1234567W

line 178 is testing for missing name prefix `n/`  while line 181 is testing for missing student_id prefix `s/`.
It is overly complicated and we can refactor if you have a better way of doing this 😄